src/common/conditional_compilation/include/openvino/cc/factory.h

```diff
@@ -31,11 +31,16 @@ class Factory<Key, T(Args...)> {
 #ifdef SELECTIVE_BUILD
 #    define registerNodeIfRequired(Module, Name, key, Impl) \
         OV_PP_EXPAND(OV_PP_CAT(registerImpl, OV_CC_SCOPE_IS_ENABLED(OV_PP_CAT3(Module, _, Name))) < Impl > (key))
-#    define createNodeIfRegistered(Module, key, ...) createImpl(key, __VA_ARGS__)
+#    define registerNodeWithImplIfRequired(Module, Name, key, Impl) \
```

Comment:
A shorter name is better. What do you think about registerImplIfRequired?

Reply:
Good suggestion, will be done!